datahub-web-react/src/app/identity/user/ViewAssignRoleModal.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Button, message, Modal, Typography } from 'antd';
+import { useBatchAssignRoleToActorsMutation } from '../../../graphql/mutations.generated';
+import { DataHubRole } from '../../../types.generated';
+
+type Props = {
+    visible: boolean;
+    roleToAssign: DataHubRole | undefined;
+    userUrn: string;
+    username: string;
+    onClose: () => void;
+    onConfirm: () => void;
+};
+
+export default function ViewAssignRoleModal({ visible, roleToAssign, userUrn, username, onClose, onConfirm }: Props) {
+    const [batchAssignRoleToActorsMutation] = useBatchAssignRoleToActorsMutation();
+    // eslint-disable-next-line
+    const batchAssignRoleToActors = () => {
+        if (roleToAssign) {
+            batchAssignRoleToActorsMutation({
+                variables: {
+                    input: {
+                        roleUrn: roleToAssign.urn,
+                        actors: [userUrn],
```

Comment:
Just one user??

Reply:
Feels like we may not need this modal at all

Reply:
Per offline discussion refactored this to be a Popconfirm!